code/modules/clothing/under/_under.dm

```diff
@@ -305,3 +305,21 @@
 
 /obj/item/clothing/under/rank
 	dying_key = DYE_REGISTRY_UNDER
+
+/obj/item/clothing/under/proc/dump_attachment()
+	if(!attached_accessory)
+		return
+	var/atom/L = drop_location()
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Reply:
Oops! Will change that.